pkg/imgpkg/imagedesc/types.go

```diff
@@ -133,3 +140,23 @@ func (t ImageOrIndex) Tag() string {
 		panic("Unknown item")
 	}
 }
+
+func (t ImageOrIndex) MountableImage(registry RegistryRemoteImage) (regv1.Image, error) {
+	if t.Image == nil {
+		return nil, fmt.Errorf("Unable to retrieve a mountable image on an imageindex")
+	}
+
+	reference, err := regname.ParseReference((*t.Image).Ref())
+	if err != nil {
+		return nil, err
+	}
+	descriptor, err := registry.Get(reference)
+	if err != nil {
+		return nil, fmt.Errorf("Getting mountable image failed: %s: %s", reference, err)
```

Comment:
out of curiosity, how do we know it's a mountable image? Get seems to be returning plain regv1.Image

Reply:
Get() returns a `regremote.Descriptor` which is metadata about an image. 
From the docs about remote Descriptors:
```
Descriptor provides access to metadata about remote artifact and accessors for efficiently converting it into a v1.Image or v1.ImageIndex.
```

We later use this remote Descriptor to get a v1.Image (which under the hood is a mountable image). 

Feels like you hinting that the word "mountable image" should be named something else?